src/e2e/step-definitions/zaak.ts

```diff
@@ -45,18 +58,10 @@ When("Employee {string} does not have enough information to finish Intake and as
 
     await this.page.waitForTimeout(10000)
     this.expect(await this.page.getByText('Document "Aanvullende informatie nodig voor zaak ZAAK-2024-0000000167" is toegevoegd aan de zaak')).toBeTruthy();
-    this.expect(await this.page.getByText(`Aanvullende informatie nodig voor zaak ${zaakNumber}`)).toBeTruthy();
-    this.expect(await this.page.getByRole('cell', { name: 'Aanvullende informatie', exact: true })).toBeTruthy()
-    this.expect(await this.page.getByRole('cell', { name: 'Assigned' })).toBeTruthy()
-    // current date to 16-02-2024 format
-    const currentDDateString = new Date().toISOString().split('T')[0].split('-').reverse().join('-');
-    this.expect(await this.page.getByRole('cell', { name: currentDDateString })).toBeTruthy()
-    this.expect(await this.page.getByRole('cell', { name: user1Profile.group })).toBeTruthy()
-    this.expect(await this.page.getByRole('cell', { name: user1Profile.username })).toBeTruthy()
+    await checkZaakAssignment.call(this, zaakNumber, user1Profile);
```

Comment:
It may be nice to move this to a independent scenario

Scenario: "Bob" wants to open smartdocuments on a newly created zaak
Given "Bob" is on a newly created zaak
When "Bob" Creates a new smart document
Then "Bob" sees the newly created task created by smartdocuments

Reply:
Still new here, so sorry for the possibly dumb question, but what of the steps you mention would the code above test?

Reply:
Asking because current change only extracts the duplicated code in a new function. However I like your proposal, as it seems we can use the default Then for the duplicating checks

Reply:
PS: I don't think we should test the entire SmartDocument flow for a number of reasons:
- this is SD and PodiumD functionality and not really part of our ZAC domain at the moment; it is more the responsibility of Dimpact (they will test entire flows through all components; we should try to focus more on ZAC itself)
- the integration with SD will be changed to something completely different in the not so far future; in which case it actually be much more part of ZAC

For now just checking if you can click on the link and not get an error is perfect I think.

Reply:
Ah i misunderstood then, i guess we can do this instead for scenario so it can fail independently
Given "Bob" is on a newly created zaak
When "Bob" clicks on Create document for newly created zaak
   And "Bob" closes the tab
Then "Bob" should not get an error

The existing code is not following it exactly, but cucumbers follows this style of test descriptions: 
    The **given** part describes the state of the world before you begin the behavior you're specifying in this scenario. You can think of it as the pre-conditions to the test.
    The **when** section is that behavior that you're specifying.
    Finally the **then** section describes the changes you expect due to the specified behavior.
source: https://martinfowler.com/bliki/GivenWhenThen.html

Reply:
Thanks @RickWoltheus! Applied your feedback